src/js/popup.js

```diff
@@ -59,15 +59,30 @@ function showNagMaybe() {
   }
 
   if (!seenComic) {
-    nag.show();
-    outer.show();
-    // Attach event listeners
-    $('#fittslaw').click(_hideNag);
-    $("#firstRun").click(function() {
-      chrome.tabs.create({
-        url: chrome.extension.getURL("/skin/firstRun.html#slideshow")
-      });
-      _hideNag();
+    chrome.tabs.query({active: true, currentWindow: true}, function (focusedTab) {
+      var firstRunUrl = chrome.extension.getURL("/skin/firstRun.html");
+
+      // Show the popup instruction if the active tab is not firstRun.html page
+      if (!focusedTab[0].url.startsWith(firstRunUrl)) {
+        nag.show();
+        outer.show();
+        // Attach event listeners
+        $('#fittslaw').click(_hideNag);
+        $("#firstRun").click(function() {
+          // If there is a firstRun.html tab, switch to the tab.
+          // Otherwise, create a new tab
+          chrome.tabs.query({url: firstRunUrl}, function (tabs) {
```

Comment:
If I have the tutorial page open in a different window, clicking on the tutorial link doesn't seem to do anything.

Reply:
Cool!

We might be going beyond what needs to happen by implementing tab/window switching logic, instead of just unconditionally opening a new tab as we do now. But it is nicer.

Reply:
The latest commit contains tab and window switching. Please take a look.